rpc_jobs/rpc_aio.yml

```diff
@@ -70,7 +84,20 @@
 
 - job-template:
     # DEFAULTS
-    STAGES: "Allocate Resources, Connect Slave, Prepare Deployment, Deploy RPC w/ Script, Setup MaaS, Verify MaaS, Install Tempest, Tempest Tests, Cleanup, Destroy Slave"
+    STAGES: >-
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Deploy RPC w/ Script,
+      Setup MaaS,
+      Verify MaaS,
+      Install Tempest,
+      Tempest Tests,
+      Prepare Kibana Selenium,
+      Kibana Tests,
```

Comment:
Looks like build 83 took a normal amount of time (3:11), and included the kibana/holland tests.